tests/benchmark/benchmark_apis_test.go

```diff
@@ -41,7 +41,7 @@ func getApi(b *testing.B, ctx context.Context, client connection.RegistryClient,
 
 func listApis(b *testing.B, ctx context.Context, client connection.RegistryClient) error {
 	b.Helper()
-	it := client.ListApis(ctx, &rpc.ListApisRequest{Parent: root()})
+	it := client.ListApis(ctx, &rpc.ListApisRequest{Parent: root().String() + "/locations/global"})
```

Comment:
Apropos of nothing, every time I see this omnipresent "/locations/global" constant in our code it sends me down a rabbit hole of wondering how we could eliminate it globally.

Reply:
I think we should do something in the names package. #470?